nautobot/users/forms.py

```diff
@@ -70,3 +73,13 @@ def clean(self):
 
 class PreferenceProfileSettingsForm(BootstrapMixin, forms.Form):
     timezone = TimeZoneFormField(required=False, help_text="Set your default timezone", widget=StaticSelect2)
+
+
+class AdminPasswordChangeForm(_AdminPasswordChangeForm):
+    def save(self, commit, *args, **kwargs):
+        # Override `_AdminPasswordChangeForm.save()` to publish admin change user password event
+        instance = super().save(commit, *args, **kwargs)
+        if commit:
+            payload = {"data": serialize_object_v2(instance)}
+            publish_event(topic="nautobot.admin.user.change_password", payload=payload)
```

Comment:
Any luck with a unit test for this one?

Reply:
ive tested it maually, but been having issues writing unittest for it